contracts/randomizer/primitive/PseudorandomAtomicBasic.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Creatd By: Art Blocks Inc.
+
+pragma solidity 0.8.17;
+
+import "../../interfaces/0.8.x/IPseudorandomAtomic.sol";
+
+/**
+ * @title PseudorandomAtomic
+ * @author Art Blocks Inc.
+ * @notice This contract atomically returns a pseudorandom value.
+ */
+contract PseudorandomAtomic {
```

Comment:
This singleton holds no state – may want to consider making this an externally linked library for deployment purposes (can discuss further offline if helpful)?

Reply:
We want to have the ability to hold state in this point of our overall contract architecture 👍

Reply:
🙏 👍 SGTM